tests/run-make/static-pie/rmake.rs

```diff
@@ -0,0 +1,76 @@
+// How to manually run this
+// $ ./x.py test --target x86_64-unknown-linux-[musl,gnu] tests/run-make/static-pie
+
+//@ only-x86_64
+//@ only-linux
+//@ ignore-32bit
+
+use std::process::Command;
+
+use run_make_support::llvm_readobj;
+use run_make_support::rustc;
+use run_make_support::{cmd, run_with_args, target};
+
+// Minimum major versions supporting -static-pie
+const GCC_VERSION: u32 = 8;
+const CLANG_VERSION: u32 = 9;
+
+// Return `true` if the `compiler` version supports `-static-pie`.
+fn ok_compiler_version(compiler: &str) -> bool {
+    let version_threshold = match compiler {
+        "clang" => CLANG_VERSION,
+        "gcc" => GCC_VERSION,
+        other => panic!("unexpected compiler '{other}', expected 'clang' or 'gcc'"),
+    };
+
+    if Command::new(compiler).spawn().is_err() {
```

Comment:
`std::process::Command` is used because `run_make_support::command::Command` calls unwrap on `spawn` in `run_unchecked`, so commands that are not found panic.